creativecloud/scripts/scripts.js

```diff
@@ -161,6 +156,27 @@ const CONFIG = {
 
 const miloLibs = setLibs(LIBS);
 
+(async function loadScript() {
+  const firstDiv = document.querySelector('body > main > div:nth-child(1) > div');
+  if (firstDiv?.classList.contains('interactive-marquee')) {
```

Comment:
Please add a specific check for change bg block.

Reply:
done